Scripts/script-CommonServerPython.yml

```diff
@@ -2,6 +2,7 @@ commonfields:
   id: CommonServerPython
   version: -1
 name: CommonServerPython
+toversion: 4.2.0
```

Comment:
What is this for?

Reply:
This is for versions before the fix, so the integrations will still work